extensions/notebook/src/common/localizedConstants.ts

```diff
@@ -84,8 +84,8 @@ export function httpRequestError(code: number, message: string): string { return
 export function msgDownloadLocation(downloadLocation: string): string { return localize('msgDownloadLocation', "Downloading to {0}", downloadLocation); }
 
 // Create Book dialog constants
-export const newGroup = localize('newGroup', "New Group");
-export const groupDescription = localize('groupDescription', "Groups are used to organize Notebooks.");
+export const newGroup = localize('newGroup', "New Book");
```

Comment:
When you get a chance it would be nice to update these to `newBook` and `bookDescription` as well

Reply:
Yup, will do 👍